tests/integration/components/scroll-to-top-test.js

```diff
@@ -1,10 +1,17 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
 
 module('Integration | Component | scroll-to-top', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
-    assert.expect(0);
+    assert.expect(1);
+
+    await render(hbs`<ScrollToTop />`);
+    
+    await click('[data-test-scroll-to-top]');
```

Comment:
Can we please test for the action that is being performed at the click of this button?
This means when the button is clicked, the page scrolls to position 0 (check the `scroll-to-top.js` for reference)
that is needed to be tested here.

Reply:
You can test the components in the following steps :
- Scroll the window a little bit using `scrollTo`
- Click the `scroll-to-top` button 
- Write an assert  to check the height that it is `0` or not.